src/components/Home/Home.tsx

```diff
@@ -1,17 +1,22 @@
+import { useState } from 'react'
 import { redditUrl } from '../../API/Constants'
 import { get } from '../../API/API'
+import { Table } from '../Table/Table'
```

Comment:
why is it `Table/Table`

Reply:
Junaid said .... avoid to use index named files. component should be a proper name.

Reply:
ok then we follow snake-case naming convention for directories

Reply:
oh... Ok